docs/okeps/okep-5494-ovn-kubernetes-mcp-server.md

```diff
@@ -0,0 +1,740 @@
+      southbound database
+    * **ovn-appctl -t ovn-controller ct-zone-list** to list all the conntrack
+      zone to OVN construct mapping for better understanding how the conntrack
+      commit of the packets happened
+* OpenvSwitch Layer - RHEL - openvswitch database is usually gathered as part
+  of sos-report for offline troubleshooting
+    * **ovs-ofctl dump-flows** to debug specially the br-ex/breth0 openflows
+      that OVN-Kubernetes creates on the gateway of each node
+    * **ovs-appctl dpctl/dump-flows** to trace live packets run on a specific
+      node's ovs container (KIND) or on the node (OpenShift)
+    * **ovs-appctl ofproto/trace** and detrace to run an ovs trace of the
+      packet based on the openflows
+    * **ovs-appctl dpctl/dump-conntrack** to know all the conntrack zones used
+      for a specific connection
+    * **ovs-vsctl** commands to list interfaces and bridges
+    * **retis** to see packet drops in ovs
+* Netfilter/Kernel Layer - RHEL - this information is usually gathered as part
+  of sos-report for offline troubleshooting
+    * **ip util commands** like **ip r** or **ip a** or **ip rule list** for
+      debugging VRFs, BGP learnt routes or the routes OVN-Kubernetes creates
+      on the node or custom routes that customer's add
+    * **nft list ruleset** to understand what rules were created by
+      OVN-Kubernetes specially in routingViaHost=true gateway mode
+    * **iptables-save** to list all iptables (Given iptables is deprecated,
+      I think we can skip this tool though for now 50% of ovn-kubernetes is
+      still on IPT)
+    * **conntrack** -L or -E on the host itself
+* TCPDUMP - external open source tools, can't be used for offline
+  troubleshooting
+    * **tcpdump** is used for packet capture and analysis
+    * **pwru** is used to know the kernel drop reason
+
+Ideally speaking, there are metrics and events that via alerts also go to
+the dashboard which is probably what most end-users use to troubleshoot.
+So when we do the phase3 we would need to reconsider this stack of
+troubleshooting entirely for including other aspects like OVN-Kubernetes
+troubleshooting dashboard that the observability team created or the
+various packet drop tools observability team already exposes. But for
+the scope of this enhancement, for now, we will consider these above set
+of tools as MVP.
+
+As we can see, that's a lot of tools! So remembering the syntax for each of
+these tools always and executing them one-by-one and gathering the information
+at each layer, analysing them, and then moving to the next layer takes time
+for a human. Always during an escalation the part that takes the longest is
+the RCA by combing through all the data - same goes for troubleshooting a
+cluster (which is slightly easier when we have access to the cluster than
+analysing offline data). The fix is usually the easiest part (there are
+exceptions).
+
+```
```

Comment:
Love the diagram! I can imagine it will be helpful to AI to know which tools to use :smile: